### PR TITLE
Updated syntax

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
#### Summary

Updated syntax as per ES6.
#### Related issues

Resolving issue #16662 
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by PrajwalBorkar** [comment]
Please review for changes.

File: files/en-us/web/api/idbobjectstore/deleteindex/index.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ DBOpenRequest.onsuccess = function(event) {
   note.innerHTML += '<li>Database initialized.</li>';
 
   // store the result of opening the database in the db variable. This is used a lot below
-  db = this.result;
+  let db = this.result;
```
Comment (teoli2003) [inline files/en-us/web/api/idbobjectstore/deleteindex/index.md:77]: No need for `let` as it already defined at line 63.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by PrajwalBorkar** [comment]
There are some files that have var but missing in the sheet and should also be updated?

**Comment by PrajwalBorkar** [comment]
There are some files which are not in sheet but contains var so I have updated that also. Please check once. Should I update the missing files in the sheet in future also?

**Comment by teoli2003** [comment]
Yes, if you find extra files, add them to the spreadsheet.

**Review by teoli2003 — APPROVED** [review]

**Comment by PrajwalBorkar** [comment]
> Yes, if you find extra files, add them to the spreadsheet.

Ok will add this one and upcoming files.

**Pull request merged**